core/DataTable/Renderer/Json.php

```diff
@@ -73,9 +73,30 @@ protected function renderTable($table)
         // silence "Warning: json_encode(): Invalid UTF-8 sequence in argument"
         $str = @json_encode($array);
 
+        if ($str === false && json_last_error() === JSON_ERROR_UTF8) {
+            $str = @json_encode(self::makeArrayUtf8($array));
```

Comment:
Maybe we shouldn't silence this second function call? Just so any other issues won't be silenced.

Reply:
makes sense 👍